src/ServiceControl.AcceptanceTests/Monitoring/CustomChecks/When_email_notifications_are_enabled.cs

```diff
@@ -0,0 +1,104 @@
+﻿namespace ServiceControl.AcceptanceTests.Monitoring.CustomChecks
+{
+    using System;
+    using System.IO;
+    using System.Linq;
+    using System.Threading.Tasks;
+    using AcceptanceTesting;
+    using Notifications;
+    using NServiceBus;
+    using NServiceBus.AcceptanceTesting;
+    using NServiceBus.CustomChecks;
+    using NUnit.Framework;
+    using ServiceBus.Management.Infrastructure.Settings;
+    using TestSupport.EndpointTemplates;
+
+    [TestFixture]
+    [RunOnAllTransports]
+    class When_email_notifications_are_enabled : AcceptanceTest
+    {
+        [Test]
+        public async Task Should_send_custom_check_status_change_emails()
+        {
+            var emailDropPath = Path.Combine(Path.GetTempPath(), Path.GetRandomFileName());
+            Directory.CreateDirectory(emailDropPath);
+            string[] emails = new string[0];
+
+            SetSettings = settings =>
+            {
+                settings.NotificationsFilter = "MyCustomCheckId#Other custom check";
+                settings.EmailDropFolder = emailDropPath;
+                settings.InitializeStore = store =>
+                {
+                    using (var session = store.OpenSession())
+                    {
+                        var notificationsSettings = new NotificationsSettings
+                        {
+                            Id = NotificationsSettings.SingleDocumentId,
+                            Email = new EmailNotifications
+                            {
+                                Enabled = true,
+                                From = "YouServiceControl@particular.net",
+                                To = "WhoeverMightBeConcerned@particular.net",
+                            }
+                        };
+
+                        session.Store(notificationsSettings);
+
+                        session.SaveChanges();
+                    }
+                };
+            };
+
+            await Define<MyContext>(c =>
+                {
+                    c.EmailDropPath = emailDropPath;
+                })
+                .WithEndpoint<EndpointWithFailingCustomCheck>()
+                .Done(c =>
+                {
+                    emails = Directory.EnumerateFiles(emailDropPath).ToArray();
+                    return emails.Length > 0;
+                })
+                .Run();
+
+
+
+            Assert.True(emails.Length > 0);
```

Comment:
Maybe use collection asserts here or use a string message to indicate what was wrong should this fail?